test-data/genops-try.test

```diff
@@ -140,20 +149,22 @@ L3: (handler for L2)
     r14 = getattr r12, r13
     r15 = py_call(r14, r11)
     r16 = cast(None, r15)
-L4:
-    clear_exc_info
+L4: (handler for L3)
+    restore_exc_info r10
+    r17 = no_err_occurred
 L5:
     goto L8
-L6: (handler for L1, L3, L4, L5)
-    r17 = err_catch
-    r18 = unicode_4 :: static  ('weeee')
-    r19 = builtins.module :: static
-    r20 = unicode_1 :: static  ('print')
-    r21 = getattr r19, r20
-    r22 = py_call(r21, r18)
-    r23 = cast(None, r22)
-L7:
-    clear_exc_info
+L6: (handler for L1, L4, L5)
+    r18 = err_catch
+    r19 = unicode_4 :: static  ('weeee')
+    r20 = builtins.module :: static
+    r21 = unicode_1 :: static  ('print')
+    r22 = getattr r20, r21
+    r23 = py_call(r22, r19)
+    r24 = cast(None, r23)
+L7: (handler for L6)
+    restore_exc_info r18
+    r25 = no_err_occurred
 L8:
-    r24 = None
-    return r24
+    r26 = None
+    return r26
```

Comment:
It would be good to have a genops test that catches an exception of a particular type.